backend/.eslintrc.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
   },
   parserOptions: {
     parser: 'babel-eslint',
-    ecmaVersion: 2015,
+    ecmaVersion: 9,
```

Comment:
??????????

Reply:
Required for Object spread syntax to not be complained about in node.